### DISCUSSION TITLE
[Question]  flet_fastapi.app() The customized resource path does not take effect.

### DISCUSSION THREAD

**Discussion by rogeremail** [flet-dev/flet#2432] [Q&A]
### Question

How can i do it?
Thank you.

### Code sample

```python
import flet as ft
import flet_fastapi

async def main(page: ft.Page):
    await page.add_async(ft.Image(src=f"favicon.png"))
    await page.update_async()

#ft.app(target=main, assets_dir="assets", view=ft.AppView.WEB_BROWSER) # This is normal.
app = flet_fastapi.app(main, assets_dir="E:/tmp/testIcon_async/assets") # This doesn't work.
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by Daxexs** [comment]
```python
from pathlib import Path

assets = Path(__file__).resolve().parent / "assets"

app = flet_fastapi.app(main, assets_dir=assets)
```

**Reply by rogeremail** [reply]
# I changed it to :

import flet as ft
import flet_fastapi
from pathlib import Path


async def main(page: ft.Page):
    await page.add_async(ft.Image(src=f"favicon.png"))
    await page.update_async()

assets = Path(__file__).resolve().parent.joinpath("assets")
print(f"----{assets}---") # ----E:\tmp\testIcon_async\assets---
app = flet_fastapi.app(main, assets_dir=str(assets))  # This doesn't work.
#ft.app(target=main, assets_dir=str(assets), view=ft.AppView.WEB_BROWSER) # It is work

**Comment by rogeremail** [comment]
Thank you very much! It's normal. I forgot to clear the cache.
